truffle/src/main/ruby/core/rubinius/common/object_space.rb

```diff
@@ -35,8 +35,7 @@ def self._id2ref(id)
     if find_object([:object_id, Integer(id)], ary) > 0
       return ary.first
     end
-
-    return nil
+  return nil
```

Comment:
Hmm now you've said that yeah it doesn't make sense. Don't worry - I'll revert it.